style.css

```diff
@@ -2134,6 +2134,20 @@ blockquote {
 .main-navigation a {
   font-family: "museo", Arial, sans-serif;
 }
+@media (min-width: 37.5em) {
+  .main-navigation #primary-menu > li:last-of-type a {
```

Comment:
Can we just attach these styles to a class. I don't think it makes sense that for these styles to apply, the button MUST be the last a, in this menu. We might want to add this button to the footer or on another page, etc.

Reply:
Can you add these button styles to forms/_buttons.scss? and could you try to use the styles that are already present for buttons -- ex. inherit the existing padding and border-radius of 3?

Reply:
I did it this way because of some quirks with WordPress - 

Once you add a new link within WordPress to the nav menu and it will pick up that style. So for instance.

I can't attach a class directly to the "a" tag in WordPress, only to the parent "li" so that is why I didn't attach this all to a class. I do agree though that it should be a button class and I think when we move away from WordPress this is the way it needs to be.  If you prefer I can setup this class now in addition to the current method.

I am fine creating _buttons.scss to contain our button styles.

Reply:
Ah, that's frustrating. Why not style the li as wordpress does? Seems better than it being determined by order. I dunno, definitely fighting WP here.